docker/e2e/chaos.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash 
```

Comment:
I wonder how easy using BATS (https://github.com/bats-core/bats-core) would help here to define tests, WDYT?

Reply:
I'm all for it, BATS is pretty simple, I just made snapshots script as well, still tweaking it, but currently generates ledger.xxxx.tar.gz & ledger_log.txt @hansl